core/templates/dev/head/domain/summary/ExplorationSummaryBackendApiService.js

```diff
@@ -25,7 +25,7 @@ oppia.factory('ExplorationSummaryBackendApiService', [
       ValidatorsService, AlertsService) {
     var _fetchExpSummaries = function(
         explorationIds, includePrivateExplorations, successCallback,
-        errorCallback) {
+        errorCallback) {//console.log(ValidatorsService.isValidExplorationId);
```

Comment:
is this expected? 
Just a suggestion, before making a pull request from GitHub you should go through the "Files Changed" tab at least once.

Reply:
@anookeen, What about this?
Also, can you make your PR title look like "Fix part of #bugnum: ....."?